azurelinuxagent/common/event.py

```diff
@@ -345,6 +337,30 @@ def add_metric(self, category, counter, instance, value, log_event=False):
         except EventError as e:
             logger.error("{0}", e)
 
+    @staticmethod
+    def _cleanup_message(message):
```

Comment:
Cleanup is a noun, so I thought this would be a message regarding cleanup :) I suggest either renaming to `clean_up_message` or `cleaned_message` or something similar.